packager/debugger.html

```diff
@@ -14,8 +14,144 @@
 <!-- Fake favicon, to avoid extra request to server -->
 <link rel="icon" href="data:;base64,iVBORw0KGgo=">
 <title>React Native Debugger</title>
+<script src="//cdnjs.cloudflare.com/ajax/libs/Chart.js/1.0.2/Chart.min.js"></script>
```

Comment:
Can you bring in Chart.min.js in the codebase, we want this to work offline as well and don't want it to die if next year the URL becomes invalid.